example-playbooks/replace_node/replace_node.yml

```diff
@@ -64,3 +64,134 @@
+      block:
+        - name: Get the host id for all nodes
+          uri:
+            url: "http://{{ scylla_api_address }}:{{ scylla_api_port }}/storage_service/host_id"
+            method: GET
+          register: _host_ids
+          until: _host_ids.status == 200
+          retries: 5
+          delay: 1
+          delegate_to: "{{ temporary_seed }}"
+
+        - set_fact:
+            _replaced_node_host_id: "{{ item.value }}"
+          when: item.key == replaced_node
+          loop: "{{ _host_ids.json }}"
+
+        - name: Set replace_node_first_boot
+          lineinfile:
+            path: /etc/scylla/scylla.yaml
+            regexp: '^(#\s*)?replace_node_first_boot:'
+            line: "replace_node_first_boot: {{ _replaced_node_host_id }}"
+            create: yes
+      when: _replace_node_first_boot_grep.failed == false
+
+    - name: Set replace_address_first_boot
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^(#\s*)?replace_address_first_boot:'
+        line: "replace_address_first_boot: {{ replaced_node }}"
+        create: yes
+      when: _replace_node_first_boot_grep.failed
+
+    - name: Start Scylla
+      service:
+        name: scylla-server
+        state: started
+
+    - name: Wait for CQL port on {{ listen_address }}
+      wait_for:
+        port: 9042
+        host: "{{ listen_address }}"
+        timeout: "{{ scylla_bootstrap_wait_time_sec }}"
+
+    - name: Wait for the added node to become healthy
+      shell: |
+        nodetool status|grep -E '^UN|^UJ|^DN'|grep "{{ vars['ansible_'~scylla_nic].ipv4.address }}"| wc -l
+      register: node_count
+      until: node_count.stdout|int == 1
+      retries: 300
+      delay: 1
+
+    - name: Make sure that the seed also sees the new node in the UN state
+      shell: |
+        nodetool status|grep -E '^UN|^UJ|^DN'|grep "{{ vars['ansible_'~scylla_nic].ipv4.address }}"| wc -l
+      register: node_count
+      until: node_count.stdout|int == 1
+      retries: 300
+      delay: 1
+      delegate_to: "{{ temporary_seed }}"
+
+    - name: Remove the replace_node_first_boot record
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^replace_node_first_boot:'
+        state: absent
+      when: _replace_node_first_boot_grep.failed == false
+
+    - name: Remove the replace_address_first_boot record
+      lineinfile:
+        path: /etc/scylla/scylla.yaml
+        regexp: '^replace_address_first_boot:'
+        state: absent
+      when: _replace_node_first_boot_grep.failed
+
+    - name: start and enable the Manager agent service
+      service:
+        name: scylla-manager-agent
+        state: restarted
+        enabled: yes
+
+    - name: Check if RBNO is available in the current Scylla version
+      shell: |
+        scylla --help | grep enable-repair-based-node-ops
+      register: _enable_rbno_grep
+      ignore_errors: true
+
+    - name: Check if RBNO is enabled for node replacement
```

Comment:
At this stage it's "if RBNO was enabled" ;)

Reply:
Fixed